src/_xoConnection.js

```diff
@@ -127,13 +135,12 @@ class XoConnection extends Xo {
     return find(await this.call("schedule.getAll"), predicate);
   }
 
-  async deleteTempResources() {
-    const disposers = this._tempResourceDisposers;
+  async deleteResources(disposers) {
```

Comment:
You can:
-  rename `deleteResources` -> `_cleanDisposers`
-  create two methods `deleteTempResources`  and `deleteDurableResources` which call it.